test/test_helper.exs

```diff
@@ -2,4 +2,4 @@ Vapor.load!([%Vapor.Provider.Dotenv{}])
 
 Mimic.copy(Joken.Signer)
 
-ExUnit.start(exclude: [skip: true, integration: true])
+ExUnit.start(exclude: [:skip, :integration, :priv_key_integration, :token_integration])
```

Comment:
could this simply just be `:integration` instead of the sub tags as well?

I like the idea of the sub tags if you only want to run parts, but what I don't want to see is this list of excludes exploding as we add more integration test tags.

Reply:
will remove the tags and revert to original

Reply:
added some notes about ROLES, which was one of the reasons I had the split in the first place.